practice.js

```diff
@@ -14,7 +15,7 @@ while(i<num){
 
 let word="madam";
 let str="";
-for(let i=word.length;i<0;i++){
+for(let i=word.length-1;i>=0;i--){

```

Comment:
hey pankaj ! next time please focus if i do any mistake and you modify that so please give me there comment using //